src/v2/System/i18n/ClassI18n.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react"
+import { TFunction, useTranslation } from "react-i18next"
+
+interface ClassI18nProps {
+  children:
+    | React.ReactNode
+    | (({ t }: { t: TFunction<"translation", undefined> }) => React.ReactNode)
+}
+
+export const ClassI18n = ({ children }: ClassI18nProps) => {
```

Comment:
@lidimayra - mind adding some docs in this file around what this file does and why its needed?